app/views/recipes/show.html.erb

```diff
@@ -1,42 +1,51 @@
 <h2><%= @recipe.title %></h2>
-
-<% unless current_user.favorites.pluck(:recipe_id).include?(@recipe.id) %>
-  <%= button_to 'Favorite', profile_favorites_path, params: { id: @recipe.id } %>
-<% end %>
-<%= image_tag @recipe.image %><br>
-<p>Prep time: <%= @recipe.time %> minutes</p>
-
-<section id="ingredients">
-  <h3>Ingredients:</h3>
-  <ul>
-    <% @recipe.ingredients.list.each do |ingredient| %>
-      <li><%= ingredient %></li>
-    <% end %>
-  </ul>
-</section>
-
-<section id="instructions">
-  <h3>Instructions:</h3>
-  <% if @recipe.instructions.list.nil? %>
-    <p>Instructions are unavailable for this recipe.</p>
-  <% else %>
-    <ol>
-      <% @recipe.instructions.list.each do |instruction| %>
-        <li><%= instruction %></li>
+<div class="container">
+  <div class="row">
+    <div class="col-sm-6" class="panel panel-default">
+      <%= image_tag @recipe.image, :class => "thumbnail", :width => "100%" %><br>
+      <p>Prep time: <%= @recipe.time %> minutes</p>
+      <% if current_user.favorites.pluck(:recipe_id).include?(@recipe.id) %>
+      <%= link_to 'Delete from Favorites', profile_favorite_path(@recipe.id), method: :delete, :class => "btn btn-default" %>
+      <% else %>
+      <%= button_to 'Add to Favorites', profile_favorites_path(id: @recipe.id), method: :post, :class => "btn btn-default" %>
       <% end %>
-    </ol>
-  <% end %>
-</section>
-
-<section id="nutrition">
-  <h3>Nutrition Information:</h3>
-  <ul>
-    <li><%= @recipe.nutrition.calories %> calories</li>
-    <li><%= @recipe.nutrition.protein %> g protein</li>
-    <li><%= @recipe.nutrition.fat %> g fat</li>
-    <li><%= @recipe.nutrition.carbohydrates %> g carbohydrates</li>
-    <li><%= @recipe.nutrition.cholesterol %> mg cholesterol</li>
-    <li><%= @recipe.nutrition.sodium %> mg sodium</li>
-    <li><%= @recipe.nutrition.sugar %> g sugar</li>
-  </ul>
-</section>
+    </div>
+  <div class="col-sm-6">
+    <section id="ingredients">
+      <h3>Ingredients:</h3>
+      <ul>
+        <% @recipe.ingredients.list.each do |ingredient| %>
+        <%= ingredient %><br>
+        <% end %>
+      </ul>
+    </section>
+  </div>
+</div>
+<div class="row">
+  <div class="col-sm-6">
+    <section id="nutrition">
+      <h3>Nutrition Information:</h3>
+        <%= @recipe.nutrition.calories %> calories<br>
+        <%= @recipe.nutrition.protein %> g protein<br>
+        <%= @recipe.nutrition.fat %> g fat<br>
+        <%= @recipe.nutrition.carbohydrates %> g carbohydrates<br>
+        <%= @recipe.nutrition.cholesterol %> mg cholesterol<br>
+        <%= @recipe.nutrition.sodium %> mg sodium<br>
+        <%= @recipe.nutrition.sugar %> g sugar<br>
+    </section>
+  </div>
+  <div class="col-sm-6">
+    <section id="instructions">
+      <h3>Instructions:</h3>
+      <% if @recipe.instructions.list.nil? %>
+        <p>Instructions are unavailable for this recipe.</p>
```

Comment:
Still an unfortunate end point of the api

Reply:
very much so